app/src/main/graphql/com/cornellappdev/android/eatery/ctEateryQuery.graphql

```diff
@@ -0,0 +1,34 @@
+query AllCtEateries {
+    collegetownEateries {
+      	address        
+        categories
+        coordinates {
+            latitude
+            longitude
+        }
+        eateryType
+        id
+        imageUrl
+        name
+        paymentMethods {
+            brbs
+            cash
+            cornellCard
+            credit
+            mobile
+            swipes
+        }
+        phone
+        price
+        rating
+        url
+        operatingHours{
+          date
+          events{
```

Comment:
Spacing on the "{" after operatingHours and events

Reply:
resolved